nano-win.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = Nano for Windows
+Version = 4.9
+License = GPLv3
+Description = Nano for Windows (nano-win) is the ported, standalone version of GNU nano, a famous, easy-to-use and handy text editor on Linux. To extract, 7-Zip or a similar tool is required. Copy the "nano.exe" file in "i686-w64-mingw32\bin\" directory to the target folder, preferably add the directory to PATH in Environment Variables.
+Category = 12
+URLSite = https://github.com/lhmouse/nano-win
+URLDownload = https://files.lhmouse.com/nano-win/nano-win_8920_v4.9-168-gccf508e2.7z
+SHA1 = c544eca9783c8b2667f38a385b39a9fabc83a7d1
+CDPath = none
+SizeInBytes = 538012
+Screenshot1 = https://imgur.com/Hzr4Eus
```

Comment:
Did you tested your added txt file within RAPPS in ReactOS? I doubt a bit whether this screenshot link would work properly.

Reply:
You were right. It did not resolve the image, so I changed that. I tested it on latest build. It works.